locales/en/translation.json

```
@@ -1,7 +1,8 @@
 {
     "$Language": {
         "en": "English",
-        "pt-BR": "Português - Brasil"
+        "pt-BR": "Português - Brasil",
+		"it": "Italiano"
```

This is using tabs instead of space, which makes the formating not really consistent. The following should fix it:

```suggestion
        "it": "Italiano"
```